region/maridia/inner-yellow/Thread The Needle Room.json

```diff
@@ -505,11 +505,17 @@
       "requires": [
         {"or": [
           "h_hasBeamUpgrade",
-          "canDodgeWhileShooting",
+          "canTrickyDodgeEnemies",
```

Comment:
I think I would only bump this up to a canTrickyJump. Mostly for knowing how to dodge the puyos and to move while shooting to have shots hit enemies reliably. Charge doesn't help a lot, but it does make it so you dont have to be moving forward to hit enemies, so i think its fine.
```suggestion
          "canTrickyJump",
```